src/App.js

```diff
@@ -25,6 +24,11 @@ class App extends Component {
       fontTitle: customData.fontTitle,
       fontMeta1: customData.fontMeta1,
```

Comment:
this is the name there was when i got the project so i did not change it,it is for the font type description etc